test/extended/builds/pipeline.go

```diff
@@ -100,6 +95,13 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 	})
 
 	g.Context("Pipeline with maven slave", func() {
+		g.AfterEach(func() {
+			if os.Getenv(jenkins.DisableJenkinsGCStats) == "" {
+				g.By("stopping jenkins gc tracking")
+				ticker.Stop()
+			}
+		})
+
```

Comment:
It's more about preventing a page of errors at the end of the each test. Ginkgo AfterEaches run in scope order, inner to outer.  By pushing this AfterEach in a scope, we ensure that it runs before the framework deletes the namespace, not after.